index.js

```diff
@@ -85,6 +85,19 @@ Mailgen.prototype.generatePlaintext = function (params) {
     // Render the plaintext theme with ejs, injecting the data accordingly
     var output = ejs.render(this.cachedPlaintextTheme, ejsParams);
 
+    // Definition of the <br /> tag as a regex pattern
```

Comment:
@vuchl My bad, we actually shouldn't do this, as we need the `ejs` statements to be parsed first (so that the foreach statements are executed), so that the `<br />` statements get output for each item we are printing.

This is what is output now if we run the replace logic in `cacheThemes()`:

```
Hi John Appleseed,

Welcome to Mailgen!
We're very excited to have you on board.
To get started with Mailgen, please click here:
https://mailgen.js/confirm?s=d9729feb74992cc3482b350163a1a010

Need help, or have questions?
Just reply to this email, we'd love to help.
Yours truly,
Mailgen

&copy; 2016 <a href="https://mailgen.js/" target="_blank">Mailgen</a>. All rights reserved.
```

Can you please undo the last commit?